external-api/api-tests/v1/geospatial-addresses.api-test.ts

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
Please remove all these inline rules.

Reply:
I reduced number of lint rules, Oscar fix can remove one more.

Solving type issues and improving typescript will take quite some time.